core/tests/protractor_utils/users.js

```diff
@@ -47,6 +53,10 @@ var logout = function() {
 // The user needs to log in immediately before this method is called. Note
 // that this will fail if the user already has a username.
 var _completeSignup = function(username) {
+  // Protractor waits for the angular variable to be present when loading
+  // a new page. Since the signup page is non-angular, this check can be
```

Comment:
Hang on -- isn't our signup page Angular based? (It's the one where you choose a username and agree to the terms, right?)

(I know the page before that is non-Angular, but that is a different page from this one.)

Reply:
You're right, I just verified! Only the page before this is non-angular.

I noticed the following errors before I made the change for the signup page:
```
[16:12:56] E/protractor - Could not find Angular on page http://localhost:9001/signup?return_url=http%3A%2F%2Flocalhost%3A9001%2F : retries looking for angular exceeded
.
.
.
Expected $[0] = Entry({ level: SEVERE, message: 'http://localhost:9001/signup?return_url=http%3A%2F%2Flocalhost%3A9001%2F - Failed to load resource: the server responded with a status of 500 (Internal Server Error)', timestamp: 1581351115991, type: '' }) to equal undefined.
```

Seeing error (1) I assumed it might be because the signup page is non-angular (which isn't the case as I later found out).

(2) says the server responded with a 500 (Internal Server Error) for the signup page.. Do you know if we can find some kind of log for this?

Reply:
Sorry, I have no idea tbh. But trying to get the contents of the server log does seem like a good approach to take.

Reply:
@ankita240796 @nithusha21 do you guys have any ideas about this?

Reply:
I think this might be happening due to a redirect as mentioned in the error messages as well that the page is not angular while a client side redirect.

Reply:
@ankita240796 Which redirect are you talking about? (From what page to what page?)

I didn't think we did any client-side redirects in this part of the flow.